index.js

```diff
@@ -97,6 +103,16 @@ class Conf {
 		} catch (_) {
 			this.store = store;
 		}
+
+		const {migrations, projectVersion} = options;
+
+		if (migrations) {
+			if (!projectVersion) {
+				throw new Error('You need to specify the `projectVersion` in order for the migrations to work!');
```

Comment:
I see that you changed this to an error which seems better. Perhaps make it a TypeError?

Reply:
Wanted to keep consistency with line 64
throw new Error('Project name could not be inferred. Please specify the `projectName` option.');

But I actually ended up not being consistent at all 😅 

Maybe I should update to:
throw new Error('Project version could not be inferred. Please specify the `projectVersion` option.');
?

Reply:
Agree.